inc/Engine/Admin/Settings/Page.php

```diff
@@ -594,6 +595,20 @@ private function assets_section() {
 			]
 		);
 
+		$delay_js_list_helper = sprintf(
+			// translators: %1$s = exclusion list, %2$s = opening </a> tag, %3$s = closing </a> tag.
+			__( 'If you have problems after activating this option, copy and paste the default exclusions to quickly resolve issues:<br><code>%1$s</code><br>Also, please check our %2$sdocumentation%3$s for a list of compatibility exclusions.', 'rocket' ),
+			implode( '<br>', DelayJSSettings::get_delay_js_default_exclusions() ),
+			'<a href="' . esc_url( 'https://docs.wp-rocket.me/article/1560-delay-javascript-execution-compatibility-exclusions/?utm_source=wp_plugin&utm_medium=wp_rocket' ) . '"  target="_blank" rel="noopener">',
```

Comment:
Use Beacon here to link to the doc instead of hardcoding the URL value, that way the doc version can change based on the language (EN/FR).

You can create a new entry in the Beacon::get_suggest method for it.